backend/core/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "telemetry"
+version = "0.3.0"
+authors = ["Parity Technologies Ltd. <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0"
+
+[dependencies]
+actix = "0.11.1"
+actix-web = { version = "4.0.0-beta.4", default-features = false }
+actix-web-actors = "4.0.0-beta.3"
+actix-http = "3.0.0-beta.4"
+bincode = "1.3.3"
+bytes = "1.0.1"
+chrono = { version = "0.4", features = ["serde"] }
+fnv = "1.0.7"
+hex = "0.4.3"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0", features = ["raw_value"] }
+thiserror = "1.0.24"
+primitive-types = { version = "0.9.0", features = ["serde"] }
```

Comment:
If this is a dependency now maybe we can use `H256` for the genesis hash instead of our own type?